src/generator/AutoRest.Go/TemplateModels/ModelsTemplateModel.cs

```diff
@@ -100,6 +103,9 @@ public ModelsTemplateModel(ServiceClient serviceClient, string packageName)
                     {
                         PagedTypes.Add(m.ReturnValue().Body, m.NextLink());
                     }
+                    if (!m.NextMethodExists(serviceClient.Methods)) {
+                        NextMethodUndefined.Add(m.ReturnValue().Body);
+                    }
```

Comment:
Yep yep
It is more obvious with these lines in the cshtml files...
[MethodTemplate](https://github.com/mcardosos/autorest/blob/32e755b0dc2f28b79dee2c68e8db9207cfd8f6b6/src/generator/AutoRest.Go/Templates/MethodTemplate.cshtml#L154)
[ModelTemplate]
(https://github.com/mcardosos/autorest/blob/32e755b0dc2f28b79dee2c68e8db9207cfd8f6b6/src/generator/AutoRest.Go/Templates/ModelTemplate.cshtml#L17)